### PR TITLE
Display rich text for Venue #174197523

### PR DESCRIPTION
**Pull request by software-project**
Fixes: https://www.pivotaltracker.com/story/show/174197523
Adds rich text option for venue description.

### CODE REVIEW

**Comment by james-wills-sf** [comment]
@software-project change looks good to me!

cc @akegan @cschlessinger looks like the text in salesforce should be updated to include a space. Just something to keep in mind when we start getting html content in prod

## Screenshot: html before this change
Screen Shot 2020-08-06 at 9 48 35 AM


## Screenshot: after this change
- 2020-08-06 at 12 13 29 PM

**Comment by james-wills-sf** [comment]
Actually, it looks like this change also adds markdown parsing to the process section. What section is that @software-project ? I don't see it mentioned in the ticket

**Comment by software-project** [comment]
@james-wills-sf I wasn't mentioned in the ticket, but I assume it should be also updated as it seems to me like it's using same field, so I thought it should look consistent. Was I wrong?

**Comment by james-wills-sf** [comment]
I guess if it's using the same field we have to do the same parsing. @akegan do you know what the process section is so I can check it in qa?

File: app/assets/javascripts/listings/components/listing/process-section.html.slim [hunk 1]
```diff
@@ -24,8 +24,7 @@ accordion-heading
             | {{::session.Date | dateSuffix}}
           span.content-group_time
             | {{::session.Start_Time}}
-        p.content-group_address.c-steel.no-margin.skiptranslate ng-if="::session.Venue"
-          | {{::session.Venue}}
+        p.content-group_address.c-steel.no-margin.skiptranslate ng-if="::session.Venue" ng-bind-html="::session.Venue | stripMostTags | nl2br"
```
Comment (akegan) [inline app/assets/javascripts/listings/components/listing/process-section.html.slim:27]: I think this is for info session venues vs open house venues. I'll ask if we want to update those too, but for now they are not being updated in salesforce

**Comment by akegan** [comment]
@james-wills-sf re: the missing space. In salesforce it actually looks just like a jira ticket or something with formatting options, so the SMEs don't have to generate the html themselves. 
image

File: app/assets/javascripts/listings/components/listing/process-section.html.slim (hunk 1 above)
Comment (akegan) [reply app/assets/javascripts/listings/components/listing/process-section.html.slim:27]: asked in slack here: https://sfinnovation.slack.com/archives/CPZHGET3R/p1596818833032800

**Comment by akegan** [comment]
ok @software-project @james-wills-sf Msol updated info sessions to have those be rich text too. Let's test those out and then should be good to merge with these changes 😄

**Comment by akegan** [comment]
@james-wills-sf @software-project is this PR ready to merge? I think the last piece was to do QA for the updated info session field that should now support rich text too

File: app/assets/javascripts/listings/components/listing/process-section.html.slim (hunk 1 above)
Comment (james-wills-sf) [reply app/assets/javascripts/listings/components/listing/process-section.html.slim:27]: Decision was to go ahead and support rich text in this as well as we might want links here in the future.

**Comment by james-wills-sf** [comment]
@akegan thanks for reminding me about this. You're right that's all we need to do. I don't even think we need to QA the new field with rich text since we don't have any yet, just to make sure there is no regression for regular text. Where on the page is the info session field displayed so I can check?

**Comment by akegan** [comment]
@james-wills-sf it should be on the sidebar too. Looks like someone already added one for [ed lee apartments](https://dahlia-full.herokuapp.com/listings/a0W0P00000HbryHUAR)? 
image

**Comment by akegan** [comment]
alright so I just did some QA and it turns out that we don't allow for most of the features of rich text, but we do allow for links!

because of our "strip most tags" modifier we don't render: bold, italics, colors, new lines, bulleted lists. 
image

**Pull request merged**